crates/aiken-project/src/blueprint/mod.rs

```diff
@@ -109,13 +123,6 @@ where
     }
 }
 
-impl Display for Blueprint<Schema> {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let s = serde_json::to_string_pretty(self).map_err(|_| fmt::Error)?;
-        f.write_str(&s)
-    }
-}
```

Comment:
I got rid of various `Display` implementation because they were just annoying and unused.